File: task_03/elixir/prime.exs
defmodule Prime do

  def check(number) do
      if number == 2 do
          true
      else
          Enum.all?(2..(number-1), &rem(number,&1) != 0)

      end
  end



  def give(n), do:  Enum.filter(1..n,&check(&1))



end

number_str = IO.gets("Enter a number: ")
number = String.trim(number_str) |> String.to_integer()


IO.inspect Prime.give(number)

File: task_03/ruby/prime.rb
def check(n)
  if n == 2
    return true
  end

  for i in 2..n-1 do
    if n % i == 0
      return false
    end
  end

  return true
end

puts "enter a number"
number = gets.chomp.to_i

if number < 2
  puts "prime number not defined"




else
  for j in 2..number do
    if check(j)
      puts "prime number is #{j}"

    end
  end
end
